doc/tutorials/convert.py

```diff
@@ -154,6 +156,51 @@ def convert_exercise2_to_markdown(nb):
             nb['cells'][i] = nbformat.v4.new_markdown_cell(source=content)
             nb['cells'][i]['metadata'] = cell['metadata']
             nb['cells'][i]['metadata']['solution2'] = 'hidden'
+            if 'id' in nb['cells'][i]:
+                del nb['cells'][i]['id']
+
+
+def convert_exercise2_to_jupyterlab(nb):
+    """
+    Walk through the notebook cells and convert exercise2 Markdown cells
+    containing fenced python code to a Jupyterlab-compatible format.
+    As of 2022, there is no equivalent of exercise2 for Jupyterlab
+    ([chart](https://jupyterlab-contrib.github.io/migrate_from_classical.html)),
+    but a similar effect can be obtained with basic HTML.
```

Comment:
Note to reviewers: to test this function, one has to pip install JupyterLab, and then manually pip upgrade a few JupyterLab dependencies that are also dependencies of the classic Jupyter Notebook and of the waLBerla codegen feature. If you would prefer not to do that, you can check the result of this code online on Binder ([direct link](https://mybinder.org/v2/gh/jngrad/espresso-binder/HEAD)).